crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -463,3 +463,41 @@ pub mod add_transaction {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
I imagine these (0.8.2) fixtures will be removed once 0.10.0 is deployed on goerli.
The following tests would then move to using files for 0.9.0, while the client tests would move to 0.10.0.
The cycle would then be repeated every time the feeder gateway api introduces changes and we have to account for those in between new cairo version being deployed to goerli and mainnet.

Reply:
We'll have to remind ourselves so that we don't forget to remove these. Other than that I like the idea.

Reply:
Well, we'll hit these when option's are getting finally removed right? Or if someone does that by trying it out, finds out why the options exist through these.